src/commands/CmdStop.cpp

```diff
@@ -50,6 +50,15 @@ int CmdStop (
   // Load the most recent interval.
   auto filter = getFilter (cli);
   auto latest = getLatestInterval (database);
+  std::set <int> ids = cli.getIds ();
+
+  // We either expect no ids, or we're operating on the most current.
+  for (auto id : ids)
+  {
+    if (id != 1)
+      throw std::string ("Stop only works on the most recent open interval. "
+                         "Perhaps you want the modify command?.");
+  }
```

Comment:
I would simplify the check to a single `ids.empty ()`.

Reply:
Done in sruffell/timewarrior@c9325f1.